mlir/utils/jenkins/Jenkinsfile

```
@@ -870,7 +879,6 @@ pipeline {
                                 sh 'ls -l'
                                 sh 'python3 ./bin/createPerformanceReports.py ${CHIP}'
                                 sh 'python3 ./bin/createGemmPerformanceReports.py ${CHIP}'
-                                sh 'python3 ./bin/createGemmPerformanceReports.py ${CHIP} CK'
```

Instead of this, can you go for `|| true` to let this fail silently?

---

Should it fail silently if CK was built ?

---

I can, but it looks a bit like a bash hack. What's the drawback in having the CK stage to write its own report?

---

... me not noticing that it's doing that x.x